ui/pages/confirmation/confirmation.js

```diff
@@ -301,6 +308,47 @@ export default function ConfirmationPage({
     setApprovalFlowLoadingText(childFlow?.loadingText ?? null);
   }, [approvalFlows]);
 
+  useEffect(() => {
+    async function fetchSafeChainsList() {
+      try {
+        const response = await fetchWithCache({
```

Comment:
It looks like `response` isn't used if `useSafeChainsListValidation` is `false` ; should it be part of that `if`?

Reply:
fixed in [9bc8291](https://github.com/MetaMask/metamask-extension/pull/21924/commits/9bc8291676f99faf288b0d07ef017c7e9cf4b2d6)